packages/studio-ui/src/web/views/FlowBuilder/common/TransitionItem.tsx

```diff
@@ -126,3 +132,12 @@ export default class TransitionItem extends Component<Props> {
     }
   }
 }
+
+const mapStateToProps = (state: RootReducer) => ({
```

Comment:
you can use the `withLanguage` HOC for this manner if you want but it's not necessary

Reply:
changed as suggested